lib/blob/blobstore.c

```diff
@@ -8045,8 +8045,8 @@ bs_snapshot_checksum_blob_open_cpl(void *cb_arg, struct spdk_blob *_blob, int bs
 	ctx->blob = _blob;
 
 	if (_blob->locked_operation_in_progress) {
-		SPDK_DEBUGLOG(blob, "blob 0x%" PRIx64 " snapshot checksum - another operation in progress\n",
-			      _blob->id);
+		SPDK_ERRLOG("blob 0x%" PRIx64 " snapshot checksum - another operation in progress\n",
+			    _blob->id);
```

Comment:
Should we use Error? It is not really a bug or error. Isn't it? 
cc @shuo-wu

Reply:
I am fine with using level `error` here since it helps us understand why the checksum calculation fails when checking the instance manager logs

Reply:
Inside blobstore, for the same reason, sometimes `ERRLOG`is used, sometimes `DEBUGLOG`, so I think it is fine to make so. At the end, this is an error because the operation doesn't start ...